src/player.cpp

```diff
@@ -9928,6 +9928,24 @@ bool player::dispose_item( item& obj, const std::string& prompt )
 
 void player::mend_item( item_location&& obj, bool interactive )
 {
+    if( g->u.has_trait( "DEBUG_HS" ) ) {
+        uimenu menu( true, "Toggle which fault?" );
```

Comment:
I'd prefer to see this string translatable (even with taking into account that it's available only with debug trait).